src/reference-manual/optimization.Rmd

```diff
@@ -15,6 +13,17 @@ has the advantage of setting its own stepsize.
 
 ## General configuration
 
+All of the optimizers have the option of including the
+the log absolute Jacobian determinant of inverse parameter transforms
+in the log probability computation.
+Without the Jacobian adjustment, optimization
+returns the maxiumum likelihood estimate (MLE),
+$\mathrm{argmax}_{\theta}\ p(y | \theta)$,
+the value which maximizes the likelihood of the data given the parameters.
+Applying the Jacobian adjustment produces the maximum a posteriori estimate (MAP),
+the maximum value of the posterior distribution,
```

Comment:
->
Applying the Jacobian adjustment produces the maximum a posteriori estimate (MAP),
that maximizes the value of the posterior density in the unconstrained space.

In general MAP estimate (the mode) is different in the unconstrained and constrained space.